.gitignore

```diff
@@ -25,3 +25,5 @@ debian/pylint
 .eggs/
 .pytest_cache/
 .mypy_cache/
+.project
```

Comment:
I believe these should go into your global `.gitignore` as they are related to your dev environment and not to the project itself.

Reply:
Thank you for the tip, I've been doing this wrong for a long time !